### DISCUSSION TITLE
Welcome to zabbix-promise Discussions!

### DISCUSSION THREAD

**Discussion by StarryShark** [sumitgoelpw/zabbix-promise#26] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by aphysics** [comment]
Hi everyone. I found an issue when i'm trying to send a number with value = 0 in **lib/util.sender(options)** the "if ( !option.value )" throws an error and 0 is a possible value. So I think it could be a solution to check first for typeOf(options.value).
